Resources/Prototypes/Entities/Mobs/Player/dragon.yml

```
@@ -149,9 +149,13 @@
 - type: entity
   parent: BaseMobDragon
   id: MobDragon
+  suffix: No role or objectives
   components:
   - type: Dragon
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)